### ISSUE TITLE
assets target missing

### ISSUE THREAD

**Issue by HongxuChen** [mmstick/cargo-deb#52]
- How to reproduce

```bash
$ cargo --version
cargo 0.26.0-nightly (1d6dfea44 2018-01-26)
$ git clone git@github.com:mmstick/cargo-deb.git
$ cd cargo-deb/example
$ cargo deb
warning: description field is missing in Cargo.toml
warning: license field is missing in Cargo.toml
   Compiling example v0.1.0 (file:///home/hongxu/src/cargo-deb/example)
    Finished release [optimized] target(s) in 0.38 secs
/home/hongxu/src/cargo-deb/example/target/debian/example_0.1.0_amd64.deb
$ dpkg-deb --contents ./target/debian/example_0.1.0_amd64.deb
drwxr-xr-x 0/0               0 2018-02-16 17:28 var/
drwxr-xr-x 0/0               0 2018-02-16 17:28 var/lib/
drwxr-xr-x 0/0               0 2018-02-16 17:28 var/lib/example/
-rw-r--r-- 0/0               6 2018-02-16 17:28 var/lib/example/1.txt
-rw-r--r-- 0/0               6 2018-02-16 17:28 var/lib/example/2.txt
-rw-r--r-- 0/0              11 2018-02-16 17:28 var/lib/example/3.txt
drwxr-xr-x 0/0               0 2018-02-16 17:28 usr/
drwxr-xr-x 0/0               0 2018-02-16 17:28 usr/share/
drwxr-xr-x 0/0               0 2018-02-16 17:28 usr/share/doc/
drwxr-xr-x 0/0               0 2018-02-16 17:28 usr/share/doc/example/
-rw-r--r-- 0/0            1159 2018-02-16 17:28 usr/share/doc/example/copyright
-rw-r--r-- 0/0             139 2018-02-16 17:28 usr/share/doc/example/changelog
```

- Expected Result
`usr/bin/example` should be inside the debian package as specified in Cargo.toml.

- Possible root cause
From the logging results of `cargo deb`, the installation procedure has already started while `cargo build --release` hasn't finished.
Another fact is that when `./target/release/example` is already there, it will be included in the package finally.

**Comment by HongxuChen** [comment]
I think there should be some warning messages for the missing target issues (also see #44 ).

**Comment by kornelski** [comment]
Thanks for the report. It is indeed silently ignoring files that are missing, since asset paths are interpreted as globs, and globs are allowed not to match. I've fixed that in 06d78afa0c8f77d686801992ae058c8dc437978b
